Src/Web/Web.Shared.Net.Tests/OperationCorrelationTelemetryInitializerTests.cs

```diff
@@ -135,7 +135,7 @@ public void InitializeDoesNotAddSourceFieldForRequestForSameComponent()
             string hashedIkey = "o05HMrc4Og8W1Jyy60JPDPxxQy3bOKyuaj6HudZHTjE=";
```

Comment:
Yes - tests need to be modified. I need to mock the service you've written to use in tests. They are intentionally left broken for now. I do not indent to merge this change without first putting in the change to use your service; at which point I'll update tests as well.